packages/cardano-services/src/Asset/CardanoTokenRegistry.ts

```diff
@@ -152,10 +162,9 @@ export class CardanoTokenRegistry implements TokenMetadataService {
         throw new ProviderError(
           ProviderFailure.ConnectionFailure,
```

Comment:
Server responding with ConnectionFailure is leaking internals and could be confusing from HTTP client's perspective. Maybe Unhealthy would be better? Not a change request since this was already here.

Reply:
Connection failure error is fine since it must already be interpreted by the `AssetProvider` to set the HTTP response as `undefined`. We could improve this by not using `ProviderError` though, given `TokenMetadataService` isn't a provider.

Reply:
Good points :+1: